### DISCUSSION TITLE
Navigation.goBack() not bubbling

### DISCUSSION THREAD

**Discussion by kevinpatel20** [react-navigation/react-navigation#10387] [Q&A]
when i nest `NavigationContainer` and i am on first screen of nested `NavigationContainer` and do `navigation.goBack()` it fails to handle this action since only 1 screen is present, but ideally this event should bubble up to parent `NavigationContainer` and nested `NavigationContainer` should be popped off

Am i doing something wrong here?

Any help will be appreciated

**Comment by satya164** [comment]
each `NavigationContainer` is independent of the others. if you want them to be able to navigate between them. then don't nest containers.
